### DISCUSSION TITLE
Idea: Add optional markdown block to top of reading group home page.

### DISCUSSION THREAD

**Discussion by jamie-reece** [ManifoldScholar/manifold#3735] [Ideas]
We (University of London Press) will shortly be running an open review process using reading groups in Manifold. We've sent out guidance and will be inviting reviewers soon, but it occurred to me it might be useful to have a text block somewhere on the reading group homepage that gives owners the option to add some contextual info to reassure users they're in the right place.

**Comment by dananjohnson** [comment]
The upcoming v9 release will feature a number of improvements to the reading group homepage, including the ability to add and arrange markdown blocks!

<img width="1230" alt="Screenshot 2025-05-14 at 4 14 30 PM" src="https://github.com/user-attachments/assets/e819a6c7-6b01-441b-955b-16ab4f24c784" />
<img width="1230" alt="Screenshot 2025-05-14 at 4 16 29 PM" src="https://github.com/user-attachments/assets/87d88808-5173-4c9b-b5cb-529239c5b626" />
